website/docs/r/org_vdc.html.markdown

```diff
@@ -96,7 +142,8 @@ The following arguments are supported:
 * `include_vm_memory_overhead` - (Optional, *v2.7+*, *vCD 9.7+*) Indicates if the Flex VDC should include memory overhead into its accounting for admission control. Required with the Flex allocation model.
 * `delete_force` - (Required) When destroying use `delete_force=True` to remove a VDC and any objects it contains, regardless of their state.
 * `delete_recursive` - (Required) When destroying use `delete_recursive=True` to remove the VDC and any objects it contains that are in a state that normally allows removal.
-
+* `default_vm_sizing_policy_id` - (Optional, *v3.0+*, *vCD 10.0+*) Set of VM sizing policy IDs.
+* `vm_sizing_policy_ids` - (Optional, *v3.0+*, *vCD 10.0+*) Default VM sizing policy ID.
```

Comment:
We need to state in the docs that both fields must be used together.

```
Error: error assigning VM sizing policies to VDC: both fields are required `vm_sizing_policy_ids` and `default_vm_sizing_policy_id`
```

Reply:
added

Reply:
Thanks, missing "be":
> This field requires `vm_sizing_policy_ids` to **be** configured together.

Reply:
@Didainius FYI, as I see your comment on a similar manner, but I don't know the context:
https://github.com/vmware/terraform-provider-vcd/pull/553#pullrequestreview-496173653

Reply:
add `be`

Reply:
> @Didainius FYI, as I see your comment on a similar manner, but I don't know the context:
> [#553 (review)](https://github.com/vmware/terraform-provider-vcd/pull/553#pullrequestreview-496173653)

Sorry. I improved the description of my problem statement.